rules/internal/utils/extension.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/jhump/protoreflect/desc"
+	apb "google.golang.org/genproto/googleapis/api/annotations"
+)
+
+// GetFieldBehavior returns a slice of FieldBehavior annotations for
+// the given field.
+func GetFieldBehavior(f *desc.FieldDescriptor) []apb.FieldBehavior {
```

Comment:
Consider to use plural for a slice result: `GetFieldBehaviors`.

Reply:
The plural of "behavior" is "behavior", and this matches how the proto is written.

Reply:
Ahhh, forgive my English knowledge. Thanks!